src/CLI/Check.php

```diff
@@ -143,8 +143,8 @@ private function getConfigFilename(InputInterface $input)
 
     private function printViolations(Violations $violations, OutputInterface $output): void
     {
-        $output->writeln('<error>ERRORS!</error>');
         $output->writeln(sprintf('%s', $violations->toString()));
```

Comment:
In my opinion, we can keep the `ERRORS` line when we start printing all the violations

Reply:
I think the `ERRORS` string is superfluous and redundant at this point, are you sure you want hold it?
In this case, do you prefer rollback also the tests (modified with my last commit) based on `ERRORS` string?

Reply:
We can try to remove it for now and understand later if it was necessary or more clear with it @ricfio